src/datagrid/DataGrid.tsx

```diff
@@ -884,12 +902,15 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     // Function to build datagrid colums
     private buildDataGridColumn(column: DataGridColumn, index: number): React.ReactElement {
         const {columnName, columnID, className, style, sort, filter, width} = column;
+        const columnHeight =
```

Comment:
If the refs are null by any chance, this will be set to null.  Do we want to set some default height instead of null ?

Reply:
@maheshkulkarni4  This ref will never be null even if there is no data in datagrid. In worst case if it is null the height will set as "auto". Please refer line no 167 at DataGridColumnResize.tsx